### ISSUE TITLE
Consider using calcite-colors instead of defining them in our Sass

### ISSUE THREAD

**Issue by alaframboise** [Esri/calcite-bootstrap#225]
We re-define the [calcite-colors](https://github.com/Esri/calcite-colors) in our Sass. It might make more sense to pull in that repo and borrow those color vars as they could change in the future and it might be more maintainable. We could do the same with calcite-icons when it is ready.

Thoughts?

@TheBlueDog

**Comment by CassidyB** [comment]
We will need to work to figure this out more in detail soon.  I have Nik coming down this month to start talking about how to better tie Bootstrap to the core Calcite Web repo so we can maintain less code.

**Comment by TheBlueDog** [comment]
Just as long as it doesn't break Calcite Bootstrap from being used stand-alone. You shouldn't have to install Calcite Bootstrap and then Calcite Colors to make it work.

**Comment by CassidyB** [comment]
Agreed, and yes, that is not the goal.  The goal would be that we can build a new version that is standalone when we update Calcite Web code.
